File: EducationApp.BusinessLogicLayer/Models/PrintingEditions/PrintingEditionModelItem.cs
using EducationApp.BusinessLogicLayer.Models.Authors;
using static EducationApp.BusinessLogicLayer.Models.Enums.Enums;

namespace EducationApp.BusinessLogicLayer.Models.PrintingEditions
{
    public class PrintingEditionModelItem
    {
        public long Id { get; set; }
        public string Title { get; set; }
        public string Description { get; set; }
        public decimal Price { get; set; }
        public CurrencyType Currency { get; set; }
        public ProductType Type { get; set; }
        public AuthorModel Authors { get; set; }

    }
}

File: ClientApp/src/app/shared/models/printing-editions/PrintingEditionModelItem.ts
import { CurrencyType } from 'src/app/shared/enums/CurrencyType';
import { ProductType } from 'src/app/shared/enums/ProductType';
import { AuthorModel } from 'src/app/shared/models/author/AuthorModel';
import { BaseModel } from 'src/app/shared/models/Base/BaseModel';

export class PrintingEditionModelItem extends BaseModel 
{
    id: number;
    title: string;
    description: string;
    price: number;
    currency: CurrencyType;
    type: ProductType;
    authors: AuthorModel;

}